Chinese/CLX/clx.md

```diff
@@ -0,0 +1 @@
+楚留香新传，应该是近几年来我看的最好的武侠片了。比起那些不要脸的做作，这部局可谓上上之作。
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.